pkg/bytecode/vm.go

```diff
@@ -20,6 +20,9 @@ var (
 	// produce an invalid result. In Golang, floating point division
 	// by zero produces +Inf, and modulo by zero produces NaN.
 	ErrDivideByZero = fmt.Errorf("%w: division by zero", ErrPanic)
+	// ErrBadRepetition is returned when the right-hand side of the array
+	// repetition operator is not an non-negative integer value.
```

Comment:
```suggestion
	// repetition operator is not a positive integer value.
```
double negatives are hard to read

Reply:
Unfortunately the term "positive" excludes zero. "non-negative integer" is the mathematical term to refer to  the integer range 0.. : https://en.wikipedia.org/wiki/Natural_number. I'll try to reword it a different way.

Reply:
yup, agree, positive is not correct because it excludes zero. The go stdlib uses that term too, e.g.: https://pkg.go.dev/math/rand#Int